README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://<EvgenBilenkiy>.github.io/js_2048_game/)
```

Comment:
Please, deploy your project and add the correct link to the PR description
![image](https://github.com/mate-academy/js_2048_game/assets/99403198/9b682a42-6f4e-4d9d-bc63-092a3a842939)
```suggestion
    - [DEMO LINK](https://EvgenBilenkiy.github.io/js_2048_game/)
```